### PR TITLE
fix: Fixed regex to accept hyphen (-) on ovms model name

### PR DESCRIPTION
**Pull request by antoniomtz**
## PR Checklist

- [ x] Added label to the Pull Request for easier discoverability and search
- [ x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**issue_number**

## Test Instructions if applicable
Delete person vehicle bike model folder
Re-download models, and run pipelines.

### CODE REVIEW

File: configs/opencv-ovms/models/2022/config_template.json
Comment (jim-wang-intel) [inline configs/opencv-ovms/models/2022/config_template.json:1]: this one also needs fix on 
 "name": "person-vehicle-bike-detection-2000",
                "base_path": "/models/person-vehicle-bike-detection-2000/FP16-INT8"

File: configs/opencv-ovms/demos/common/python/openvino/model_zoo/model_api/adapters/ovms_adapter.py [hunk 1]
```diff
@@ -61,7 +61,7 @@
         if not isinstance(target_model, str):
             raise TypeError("--model option should be str")
         # Expecting format: <address>:<port>/models/<model_name>[:<model_version>]
-        pattern = re.compile(r"(\w+\.*\-*)*\w+:\d+\/models\/\w+(\:\d+)*")
+        pattern = re.compile(r"(\w+\.*\-*)*\w+:\d+\/models\/(\w+\.*\-*)+(\:\d+)*")
```
[the remark this answers is not in the document]
Comment (brian-intel) [reply configs/opencv-ovms/demos/common/python/openvino/model_zoo/model_api/adapters/ovms_adapter.py:64]: Looks good once this issue is resolved

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by vli11 — APPROVED** [review]
LGTM

**Pull request merged**